# Custom message type for polygon calculation (without virtual image transformation)
float32[] features
float32[] barycenter_features
int32 d
int32 f
float32 custom_sigma
float32 custom_sigma_square
float32 custom_sigma_square_log
float32 tangent
float32 angle_radian
float32 angle_deg
